### PR TITLE
feat: update gm tutorial to use kurtosis

### PR DESCRIPTION
**Pull request by MSevey**
## Overview

Blocked on https://github.com/rollkit/gm/pull/110

Now that the GM rollup and frontend can be run all at once with the kurtosis package we can merge the two tutorials. 

## Summary by CodeRabbit

- **Documentation**
  - Updated the GM world tutorial to use Kurtosis for managing services.
  - Revised commands for launching and interacting with the GM rollup using Kurtosis.

### CODE REVIEW

**Review by jcstein — APPROVED** [review]
utACK 🔥

**Review by Manav-Aggarwal — APPROVED** [review]

File: tutorials/gm-world.md [hunk 1]
```diff
@@ -29,71 +29,96 @@ import constants from '../.vitepress/constants/constants.js'
 
 ## 🛠️ Dependencies {#dependencies}
 
-If you followed the [quick start guide](/tutorials/quick-start), you should have the Rollkit CLI and Golang installed already. If not, here's the script for you:
+As we moved into more advanced use cases we use [kurtosis](https://docs.kurtosis.com/) to help with managing all the services we need to run. You can [install kurtosis here](https://docs.kurtosis.com/install).
```
Comment (tuxcanfly) [inline tutorials/gm-world.md:32]:
```suggestion
As we move into more advanced use cases we use [kurtosis](https://docs.kurtosis.com/) to help with managing all the services we need to run. You can [install kurtosis here](https://docs.kurtosis.com/install). 
```

**Review by tuxcanfly — DISMISSED** [review]
OK. Minor typo.

**Comment by MSevey** [comment]
@coderabbitai resolve

**Comment by MSevey** [comment]
@coderabbitai ignore

**Review by Manav-Aggarwal — APPROVED** [review]

**Pull request merged**